### PR TITLE
Updated Kotlin dependency to v1.2.30

### LINKED ISSUE #29: Update to Kotlin 1.2.30

### PR DESCRIPTION
**Pull request by thodorisbais**
Also tested it locally:

qljgzyyf2m

Resolves #29 .

### CODE REVIEW

File: generators/server/templates/pom.xml.ejs [hunk 1]
```diff
@@ -760,7 +760,7 @@
             <scope>test</scope>
         </dependency>
         <%_ } _%>
-
```
Comment (sendilkumarn) [inline generators/server/templates/pom.xml.ejs]: can remove this empty line

Comment (thodorisbais) [reply generators/server/templates/pom.xml.ejs]: Then we should also remove the other two empty lines @ L769 and L775, right?

**Comment by sendilkumarn** [comment]
can you rebase with the master branch ?

**Comment by thodorisbais** [comment]
Sure, Sendil, will do tonight.

**Comment by thodorisbais** [comment]
Ready.

**Comment by sendilkumarn** [comment]
@thodorisbais awesome. One minor comment. Can you remove the package-lock.json please?

**Comment by thodorisbais** [comment]
@sendilkumarn done

**Comment by sendilkumarn** [comment]
Thanks and congrats on your first contribution 🎉

**Comment by thodorisbais** [comment]
My pleasure! I wanna do more!

**Comment by sendilkumarn** [comment]
please go ahead :)

**Pull request merged**